test/Node/ChildrenIterationTest.php

```diff
@@ -94,5 +94,10 @@ public function testCallingNextAfterIterationShouldNotThrowException()
             // do nothing - just iterate
         }
         $nodes->next();
+        // The "pass" expectation here is just that no exception was thrown.
+        // Getting to this point in the code is the "definition of pass".
+        // phpunit will flag this as a risky test if we do not assert anything,
+        // so assert something.
+        $this->assertIsObject($nodes);
```

Comment:
Perhaps we should assert on the return value of `$nodes->current()` instead? (`false` or such?)

Reply:
In PR #13 I have done:
```
$this->assertInstanceOf(\Laminas\Ldap\Node::class, $nodes->current());
```